cypress/integration/article_spec.js

```diff
@@ -55,4 +55,85 @@ describe('News Article', () => {
       secondElement.should('have.attr', 'rel', 'dns-prefetch');
     });
   });
+
+  it('should load the Reith font', () => {
+    cy.server();
+    cy.visit('/article/article-id');
```

Comment:
this line can be removed as we do this before test - https://github.com/bbc/simorgh/pull/257/files#diff-d17953029a49f35afbe27a351894e827R11